docs/eventing/samples/kafka/README.md

```diff
@@ -0,0 +1,26 @@
+The following examples will help you understand how to use the different Apache Kafka components for Knative.
+
+## Prerequisites
+
+All examples require:
+
+- A Kubernetes cluster with
+  - Knative Eventing v0.9+
+  - Knative Serving v0.9+
+- An Apache Kafka cluster
+
+If you want to run the Apache Kafka cluster on Kubernetes, the simplest option is to install it by using Strimzi. Check out the [Quickstart](https://strimzi.io/quickstarts/) guides for both Minikube and Openshift. You can also install Kafka on the host.
+check out their [Quickstart](https://strimzi.io/quickstarts/) for both Minikube
+and Openshift guides. You can also install Kafka on the host.
+
+> Note: The `KafkaSource` is not limited to Apache Kafka installations on
```

Comment:
See other comment. Lines 16-18 should be removed if this installation scenario is not supported in this guide.